lib/hashie/mash.rb

```diff
@@ -59,7 +59,7 @@ module Hashie
   #   mash.author_.name = "Michael Bleigh"  (assigned to temp object)
   #   mash.author # => <Mash>
   #
-  class Mash < Hash
+  class Mash < Hash # rubocop:disable Metrics/ClassLength
```

Comment:
For rubocop just run `rubocop -a ; rubocop --auto-gen-config`. Or just disable Metrics/ClassLength altogether in .rubocop_config.yml.